README.md

```diff
@@ -84,11 +84,41 @@ Install all the required packages:
   
 > pip3 install -r requirements.txt  
 
+### Windows using VS Code
+
+Either download the zip of the repository sources or use git:  
+  
+> git clone https://github.com/TheAntTeam/TheAntFarm.git  
+
+In VS Code
+
+* File -> Open Folder -> open the TheAntFarm Folder
+
+* Terminal -> New Terminal (This will probably open a Powershell, which is what we will use)
+
+In the Terminal:
+
+> python -m venv .\venv  (You may have to install python3 globaly)
+
+* Restart VS code
+  
+Activate the virtual environment:  
+  
+> .\venv\Scripts\activate
+
+* Open the_ant_farm.py, in the bottom right of the interface it should say "Python", next to it should be the python verion number folled by 'venv'/venv, if not click it and change the interpreter.
+  
+Install all the required packages:
+
+> "path"\TheAntFarm\venv\Scripts\python.exe -m pip install --upgrade pip
+  
+> pip3 install -r requirements.txt  
+
 ## Running the software  
   
 The software must be run from the project directory, after the virtual environment has been activated (see installation paragraph), using the following command:
 
-> python3 the_ant_farm.py  
```

Comment:
We prefer python3 because someone could have both python 2 and python 3 versions installed.

Reply:
When you make the virtual environment with python3 in vs code it doesn't have a python3 command. Just python, if you use the python3 command it uses the global python install and doesn't find the venv libraries
![Screenshot 2022-04-05 163636](https://user-images.githubusercontent.com/11468618/161844748-2b9a02c7-aa6a-4e09-962c-63af4edb82cc.png)
.

Reply:
Python3 needs to be set as the environment in the lower right.
![Screenshot 2022-04-05 163821](https://user-images.githubusercontent.com/11468618/161845028-4d531bee-656b-415d-ab3c-c8afe83553d3.png)